andela/bp-esa-frontend
# 165379468 Fix icons in automation statistics

- fix icons on automation statistics

#### What does this PR do?
- Fixes icons in automation statistics

#### Description of Task to be completed?
- Fixing icons in the automation statistics section

#### How should this be manually tested?
- Start the application
- Login
- View the automation statistics section
The icons for each should render properly.

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
[#165379468](https://www.pivotaltracker.com/n/projects/2197045/stories/165379468)

#### Postman Documentation
- N/A

#### Screenshots (If applicable)
<img width="1428" alt="Screen Shot 2019-04-16 at 16 12 17" src="https://user-images.githubusercontent.com/38077368/56212335-78b99300-6062-11e9-8047-06d02b709eaa.png">
